test/run_test

```diff
@@ -872,19 +875,19 @@ run_pgaudit_test()
   wait_ready "$name"
 
   # enable the pgaudit extension
-  docker exec -i $(get_cid "$name") bash -c psql <<EOSQL
+  docker exec -i $(get_cid "$name") bash -c "psql <<EOSQL
```

Comment:
Hehe, what a hack :-).  Don't you want to add inline coment with the reason why the
heredoc separator went into the string? (rhbz# reference at least).

Reply:
Good idea, done.